website/draft_site/drafts/templates/drafts/autobuild.html

```diff
@@ -0,0 +1,27 @@
+View Seat:
+<!-- Human drafter is always seat 0 -->
+<a href="/draft/{{ draft_id }}">0 (You)</a>
+{% for seat in bot_seat_range %}
+    <a href="/draft/{{ draft_id }}/seat/{{ seat }}">{{ seat }}</a>
+{% endfor %}
+<br>
+<br>
+
+<b>Build</b>
+<br>
+Synergy edges: {{ num_edges }}
+<br>
+Average card power: {{ avg_power }}
+<div>
```

Comment:
This seems fine for now, but I think this manually written explanation of a bot's thought process for building is a likely candidate for debt; as soon as the model the bots use to draft changes, this becomes misleading. Ideally, a bot would be capable of outputting a detailed view of its build scoring criteria in some sort of KVP format that could be dumped here as a table. I did a similar thing for route optimization at Thriftbooks, so I can take a crack at it if you agree it's valuable.

Reply:
These are more the metrics that we use to evaluate the quality of the draft + build, rather than an explanation. They're meaningful independent of the implementations, and help evaluate the impact of changes to the implementations. They're also calculated after everything is done, so the draft / build implementations are black boxes.

These metrics might change over time or become expanded as we better understand how to measure deck quality, but I don't think they add tech debt - we need some kind of simple bottom-line metrics.

I think the kind of thing you're talking about might be useful but I'd like to hear more details / an example, including details on what would be outputted and what the interface to this logging system would be.

Reply:
Cool, I'll take some time to familiarize myself with the code base and get back to this.